src/api/queues.js

```diff
@@ -139,6 +140,7 @@ router.patch(
       message: data.message !== null ? data.message : undefined,
       messageEnabled:
         data.messageEnabled !== null ? data.messageEnabled : undefined,
+      isConfidential: req.body.isConfidential === true,
```

Comment:
This isn't working right; if I toggle the staff message on a queue, it resets it to being non-confidential. I think once a queue is created as confidential, we don't want to ever allow that to change in the future. We should have a test for this. Let's add a test for the creation endpoint too!